packages/docusaurus-plugin-content-blog/src/authors.ts

```diff
@@ -26,6 +54,12 @@ const AuthorsMapSchema = Joi.object<AuthorsMap>()
       imageURL: URISchema,
       title: Joi.string(),
       email: Joi.string(),
+      socials: Joi.object({
+        twitter: Joi.string(),
+        github: Joi.string(),
+        linkedin: Joi.string(),
+        stackoverflow: Joi.string(),
+      }).custom(normalizeSocials, 'Normalize social media URLs'),
```

Comment:
Maybe we'd want to have `.unknown()` to allow users to pass extra social platforms?

We could render them with 🔗 or 🌐 icon by default?

Note your type already supports extra attributes (`[key: string]: string;`) so validation should do it as well

And unit tests should ensure we reject things that we don't want, like `{twitch: {xyz: 42}}`. I think you need `Joi.object().pattern()` for that.